portal-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -1,8 +1,13 @@
-const { param } = require('express-validator');
+const { check } = require('express-validator');
```

Comment:
Why have we changed to check from param? BankId is only a path param and this could pass with a non path parameter now

Reply:
Not sure why I changed it to `check` but `param` is the correct choice